docs/content/en/docs/install/_index.md

```diff
@@ -19,6 +19,7 @@ or as part of an existing production cluster.
 1. Understand the [Software versions and resources](reqs.md)
    that are required
 1. [Bring or create your Kubernetes cluster](k8s.md)
+1. [Replace the default cert-manager](cert-manager.md) (optional)
```

Comment:
I would add this after this list of steps in a sort of footnote. Just to highlight more the fact that this is an optional step

Reply:
But this list is supposed to be in order.  Don't they need to replace the cert-manager before they install KLT?